cmd/neofs-cli/modules/netmap/nodeinfo.go

```diff
@@ -16,6 +16,7 @@ var nodeInfoCmd = &cobra.Command{
 	Use:   "nodeinfo",
 	Short: "Get target node info",
 	Long:  `Get target node info`,
+	Args:  cobra.NoArgs,
 	Run: func(cmd *cobra.Command, args []string) {
```

Comment:
And here? I'm not sure these `Run` changes belong to the same commit though (maybe they are, since we're about arguments here).

Reply:
seems like their removal should be a part of https://github.com/nspcc-dev/neofs-node/commit/b134dbbe02bfb804d3a2c80e07298f0f308dff4b but the linter wasn't that good if we are speaking about func literals...